src/libasr/pass/intrinsic_function_registry.h

```diff
@@ -3680,6 +3749,8 @@ namespace IntrinsicScalarFunctionRegistry {
     static const std::map<int64_t,
         std::tuple<impl_function,
                    verify_function>>& intrinsic_function_by_id_db = {
+        {static_cast<int64_t>(IntrinsicScalarFunctions::Input),
+            {nullptr, &Input::verify_args}},
```

Comment:
You need to provide the instantiate_Input here. Check how other functions are implemented and simply follow them.

Reply:
Done! :+1: Thanks a lot for this @Shaikh-Ubaid! I recall setting this to nullptr when I had started the work on the function. I had totally forgotten about it. I am sorry for all the trouble.